modular_framework_core/spawners/manage_object.py

```diff
@@ -129,10 +145,10 @@ def _delete_object(self):
     parser = argparse.ArgumentParser()
     parser.add_argument('-d', '--delete', choices=["False", "false", "true", "True"],
                         help='Use this flag if you want to delete an object')
-    parser.add_argument('-p', '--position', default="0,0,0",
-                        help='Block position, use as three floats separated by commas only')
-    parser.add_argument('-o', '--orientation', default="0,0,0",
-                        help='Block orientation (rpy), use as three floats separated by commas only')
+    parser.add_argument('-p', '--position', type=triplet_floats_to_list, default=[0, 0, 0],
```

Comment:
any reason you use custom argparse type instead of just doing multiple nargs?

https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse